src/shared/utilities/textUtilities.ts

```diff
@@ -235,3 +235,51 @@ export function getRelativeDate(from: Date, now: Date = new Date()): string {
     // if all conditionals above have failed, we're looking in terms of a > 1 year gap
     return rtf.format(Math.floor(fromAdj.getUTCFullYear() - now.getUTCFullYear()), 'year')
 }
+
+/**
+ * Format for rendering readable dates.
+ *
+ * Same format used in the S3 console, but it's also locale-aware.
+ * This specifically combines a separate date and time format
+ * in order to avoid a comma between the date and time.
+ *
+ * US: Jan 5, 2020 5:30:20 PM GMT-0700
+ * GB: 5 Jan 2020 17:30:20 GMT+0100
+ */
+export function getFormattedLocalizedDate(d: Date = new Date()): string {
```

Comment:
Don't need to explain so much in function names. It's often counterproductive. 

```suggestion
export function toLocalizedDate(d: Date = new Date()): string {
```

Reply:
I can apply name changes after integ tests run since these shouldn't change the results.